res/prometheus/bridges.c

```diff
@@ -116,12 +117,18 @@ static void bridges_scrape_cb(struct ast_str **response)
 		return;
 	}
 
-	bridge_metrics = ast_calloc(ARRAY_LEN(bridge_metric_defs) * num_bridges, sizeof(*bridge_metrics));
-	if (!bridge_metrics) {
+	metrics = ast_calloc(ARRAY_LEN(bridge_metric_defs) * num_bridges, sizeof(*metrics));
+	if (!metrics) {
 		ao2_ref(bridges, -1);
 		return;
 	}
 
+	bridge_metrics = ast_calloc(ARRAY_LEN(bridge_metric_defs), sizeof(bridge_metrics));
+	if (!bridge_metrics) {
+		ao2_ref(bridges, -1);
+		ast_free(metrics);
```

Comment:
Minor: Maybe swap these two lines to free in the reverse order that we allocated?

Reply:
Done. I also applied this order at the end of the function.